ClusterTest/src/main/scala/net/snowflake/spark/snowflake/testsuite/LowMemoryStressSuite.scala

```diff
@@ -76,13 +76,25 @@ class LowMemoryStressSuite extends ClusterTestSuiteBase {
     // Convert RDD to DataFrame
     val df = sparkSession.createDataFrame(testRDD, schema)
 
+    // Write to snowflake
+    df.write
+      .format(TestUtils.SNOWFLAKE_NAME)
+      .options(TestUtils.sfOptionsNoTable)
+      .option("dbtable", test_big_partition)
+      .mode(SaveMode.Overwrite)
+      .save()
+
+    log.info(
+      s"""Finished the first multi-part upload test ======================""".stripMargin)
```

Comment:
I prefer not including `=====` in the log message

Reply:
"====" is included just because it is there when Zihan developed this case. I just get it back. I agree it is not necessary. I will remove it.